onnx_tf/opr_checker.py

```diff
@@ -13,6 +14,7 @@
 from onnx_tf.pb_wrapper import TensorflowNode
 
 warnings.filterwarnings('ignore')
+reload(logging)
```

Comment:
BTW, what reload logging for?

Reply:
The reload clears possibly configuration by other modules and makes sure the logging setting in our code is actually used, in this case level=logging.DEBUG. In my test, the logging level in use is always the default WARNING if there is no reload.